src/read.c

```diff
@@ -1499,6 +1499,41 @@ static void ReadFuncExprBody(
 }
 
 
+/****************************************************************************
+**
```

Comment:
Being super picky, maybe mention nams already contains the list if functiin arguments, and either mention narg==LEN_LIST(nams), or drop it and recalculate it.
?

Reply:
Actually, I didn't think about this at all. Good catch! I'll drop the `narg` argument.

Reply:
Done.